docs/reference/docker_compose_alpha.yaml

```diff
@@ -0,0 +1,15 @@
+command: docker compose alpha
+short: Experimental commands
+long: Experimental commands
+pname: docker compose
+plink: docker_compose.yaml
+cname:
+    - docker compose alpha watch
+clink:
+    - docker_compose_alpha_watch.yaml
+deprecated: false
+experimental: false
+experimentalcli: false
```

Comment:
Perhaps we need to set the `experimentalcli` annotation for this. We removed the "gate" for disabling/hiding those commands, but I guess we could still use the annotation to be used on docs.docker.com. (happy to discuss / brainstorm)